src/api/wayfire/output-layout.hpp

```diff
@@ -105,6 +105,13 @@ class output_layout_t : public wf::signal_provider_t
      */
     wf::output_t *get_next_output(wf::output_t *output);
 
+    /**
```

Comment:
I think that this function should not be added to core. In retrospect, the `get_next_output` function doesn't belong here either and will be removed at some point in the future. Plugins that want to get the next/previous output can just get the list of outputs and do the search themselves.